src/main/resources/db/migration/V1.0__create_dummy_data_table.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE dummy_data
     PRIMARY KEY (id)
 );
 
-INSERT INTO dummy_data (id, name, latitude, longitude)
```

Comment:
Can you move this INSERT and the UPDATE into the script as well? Reason: In the future when we have data from the hub, we'll do a dump with the data file rather than running all these commands